blocks/hero/hero.css

```diff
@@ -1,37 +1,150 @@
 .hero-container .hero-wrapper {
-  max-width: unset;
-  padding: 0;
+    max-width: unset;
+    padding: 0;
+    max-height: 300px;
 }
 
-.hero {
-  position: relative;
-  padding: 40px 24px;
-  min-height: 300px;
-}
+.hero.block {
+    position: relative;
+    padding: 10px 0;
+    color: white;
+    font-size: 1em;
 
-.hero h1 {
-  max-width: 1200px;
-  margin-left: auto;
-  margin-right: auto;
-  color: var(--background-color);
-}
+    & .title {
+        text-shadow: 1px 3px 4px rgba(0 0 0 / 30%);
+        font-size: 28px;
+        font-family: var(--heading-font-family);
+        font-weight: 900;
+        font-style: italic;
+        line-height: 1;
+    }
 
-.hero picture {
-  position: absolute;
-  z-index: -1;
-  inset: 0;
-  object-fit: cover;
-  box-sizing: border-box;
-}
+    & .grid-container {
+        display: flex;
+        min-height: 280px;
+        margin: auto;
+        align-items: center;
+
+        & .column {
+            flex: 1;
+            padding: 10px;
+            position: relative;
+            height: auto;
+
+            & .discount-circle {
+                border-radius: 50%;
+                background-color: #ee3123;
+                border: 5px solid white;
+                font-size: 14px;
+                text-align: center;
+                line-height: 1.1em;
+                font-family: "Open Sans", sans-serif;
+                font-optical-sizing: auto;
+                font-weight: 600;
+                font-style: normal;
+                float: right;
+
+                & sup {
+                    font-size: 26px;
+                }
+
+                & p {
+                    margin-top: 0;
+                }
+
+                & sub {
+                    font-size: 9px;
+                }
+            }
+
+            & .discount-circle-size {
+                width: 130px;
+                height: 130px;
+                padding-top: 10px;
+            }
+        }
+    }
+
+    img {
+        object-fit: cover;
+        object-position: 50%;
+        width: 100%;
+        height: 100%;
+    }
 
-.hero img {
-  object-fit: cover;
-  width: 100%;
-  height: 100%;
+    picture {
+        position: absolute;
+        z-index: -1;
+        inset: 0;
+        object-fit: cover;
+        object-position: 50%;
+        box-sizing: border-box;
+        overflow: hidden;
+        background-repeat: no-repeat;
+        background-position: 50% 50%;
+        background-size: cover;
+        background-color: #388ce1;
+    }
 }
 
 @media (width >= 900px) {
-  .hero {
-    padding: 40px 32px;
-  }
+    .hero.block {
+        position: relative;
+        padding: 40px 24px;
+        min-height: 300px;
+        color: white;
+        font-size: 1em;
+
+        & .title {
+            text-shadow: 1px 3px 4px rgba(0 0 0 / 30%);
+            font-family: var(--heading-font-family);
```

Comment:
You don't have to redefine anything that's specified at a smaller screen size. Since that size still applies (as it's the default), then those values will carry up to the larger view ports.